src/game/client/w3ddebugdisplay.cpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3ddebugdisplay.h"
+#include "displaystringmanager.h"
+#include "displaystring.h"
+
+// temp location
+int GameMakeColor(unsigned char red, unsigned char green, unsigned char blue, unsigned char alpha)
```

Comment:
a home for this needs to be figured out

Reply:
I believe an equivalent function already exists in colorspace.h but I didn't know the name for it so just made one up.